charts/presidio/templates/analyzer-deployment.yaml

```diff
@@ -26,10 +26,10 @@ spec:
         - containerPort: {{ .Values.analyzer.service.internalPort }}
         resources:
           requests:
-            memory: "2000Mi"
+            memory: "1500Mi"
             cpu: "1500m"
           limits:
-            memory: "5000Mi"
+            memory: "3000Mi"
```

Comment:
I think you also need to specify in the docs min requirements

Reply:
added a note about cluster planning in install.md (https://github.com/microsoft/presidio/blob/spacy_load_twice/docs/install.md) and a note about adding more recognisers which may add to consumed mem\cpu in custom_fields.md (https://github.com/microsoft/presidio/blob/spacy_load_twice/docs/custom_fields.md). do you think it need more?